client/src/pages/ScanStore.tsx

```diff
@@ -57,20 +57,21 @@ function ScanStore() {
       cartItems.push(items[i]);
     }
     // Should do batched fetch with list of barcodes in 1 request-response
-    const item_barcodes = items.map((zippedItem: any) => zippedItem.barcode);
+    const itemBarcodes = items.map((zippedItem: any) => zippedItem.barcode);
```

Comment:
I have been seeing inconsistency on naming convention. 
It seems some variables are using camcelCase and some is in snake_case.
Could you double check with Yiheng about the naming convention used in this project?

Reply:
I have double checked and it seems most naming is in camelCase. Then this change should be good to go. You can double check the codebase and rename the variables in snake_case to camel case in following commits.